styles.css

```diff
@@ -0,0 +1,165 @@
+@import "https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,900;1,300&display=swap";
+
+* {
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  background-color: #1f2937;
+  color: white;
+  width: 100%;
+}
+
+h3 {
+  margin-left: 10%;
+  display: flex;
+  align-self: center;
+}
+
+#header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-links {
+  margin-right: 150px;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+  gap: 25px;
+}
+
+a {
+  color: rgb(211 204 204);
+  text-decoration: none;
+}
+
+#header2 {
+  display: flex;
+  justify-content: space-around;
+}
+
+.header2-left {
+  display: flex;
+  flex-direction: column;
+  width: 30%;
+  padding-left: 60px;
+}
+
+h1 {
+  font-size: 48px;
+  margin-bottom: 0;
+  color: #f9faf8;
+}
+
+li {
+  font-size: 18px;
+  color: #e5e7eb;
+}
+
+.image-placeholder {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: rgb(102 114 124);
+  width: 500px;
+  height: 250px;
+  margin: 70px 85px 100px;
+}
+
+button {
+  background-color: #3882f6;
+  border-radius: 1rem;
+  padding: 15px 32px;
+  width: 150px;
+}
+
+.info-header-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.info-header {
+  font-size: 36px;
+  font-weight: 900;
+}
+
+img {
+  width: 100px;
+  height: 100px;
+  border-radius: 8px;
+  border: 3px solid #000;
+}
+
+#image {
+  display: flex;
+  justify-content: center;
+  gap: 50px;
```

Comment:
this gap should not be used because it does not respect the space of the webpage, its only after the specified gap given to it. You should use flexbox for this one